src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -785,6 +785,21 @@ impl HummockEventHandler {
                         error!("unable to send get_min_uncommitted_sst_id result: {:?}", e);
                     });
             }
+            HummockEvent::RegisterVectorWriter {
+                table_id,
+                init_epoch,
+            } => self.uploader.register_vector_writer(table_id, init_epoch),
```

Comment:
We can only have one vector writer per table_id, right? In other words, the vector wrtier is single parallelism.